### PR TITLE
add CreationInfo to all Elements in json-ld examples

### PR DESCRIPTION
**Pull request by armintaenzertng**
CreationInfo has been made mandatory again in #401, so the examples have to be updated.

This uses the de-duplication method of extracting the CreationInfo as a blank node.

### CODE REVIEW

**Review by goneall — APPROVED** [review]
Thanks @armintaenzertng  - I like the approach of using the blank node.

File: serialization/json_ld/examples/converted_from_spdx_2.json [hunk 1]
```diff
@@ -6,43 +6,48 @@
...
+      ],
+      "dataLicense": "https://spdx.org/licenses/CC0-1.0",
+      "comment": "This is the SPDX-2.3 JSON example converted to SPDX-3.0. As there is currently no closure on how to treat licenses, they are omitted here for now."
+    },
     {
       "type": "Tool",
       "spdxId": "spdx-example:SPDXRef-Actor-LicenseFind-1.0",
-      "name": "LicenseFind-1.0"
+      "name": "LicenseFind-1.0",
+      "creationInfo": {"@id": "_:creationInfo1"}
```
Comment (maxhbr) [inline serialization/json_ld/examples/converted_from_spdx_2.json:33]: Is there a reason, why you do not use this shorthand?
```suggestion
      "creationInfo":  "_:creationInfo1"
```

Comment (armintaenzertng) [reply serialization/json_ld/examples/converted_from_spdx_2.json:33]: Thanks for the nudge, I realized that we have to change the context to have `"@type": "@id"` for `creationInfo` in order for this to work.

**Review by maxhbr — APPROVED** [review]
LGTM

**Pull request merged**